### PR TITLE
removed mobile app svc deployment

### PR DESCRIPTION
**Pull request by dcaro**
## Purpose
Remove deployment of mobile web app not used anymore.

## Does this introduce a breaking change?
```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[X] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

File: provision-team/setup.sh [hunk 1]
```diff
@@ -249,8 +245,10 @@ bash ./git_fetch.sh -u https://github.com/Azure-Samples/openhack-devops-team -s
 echo "6-Deploy ingress  (bash ./deploy_ingress_dns.sh -s ./test_fetch_build -l $resourceGroupLocation -n ${teamName}${teamNumber})"
 bash ./deploy_ingress_dns.sh -s ./test_fetch_build -l $resourceGroupLocation -n ${teamName}${teamNumber}
 
-echo "7-Provision SQL & Mobile App  (bash ./provision_sql_mobileapp.sh -s ./test_fetch_build -g $resourceGroupTeam -l $resourceGroupLocation -q $sqlServerName -m $mobileAppName -h $hostingPlanName -k $keyVaultName -u $sqlServerUsername -p $sqlServerPassword -d $sqlDBName)"
-bash ./provision_sql_mobileapp.sh -g $resourceGroupTeam -l $resourceGroupLocation -q $sqlServerName -m $mobileAppName -h $hostingPlanName -k $keyVaultName -u $sqlServerUsername -p $sqlServerPassword -d $sqlDBName
+########### TO UPDATE ###########
```
Comment (dtzar) [inline provision-team/setup.sh]:
```suggestion
```

File: provision-team/setup.sh [hunk 2]
```diff
@@ -249,8 +245,10 @@ bash ./git_fetch.sh -u https://github.com/Azure-Samples/openhack-devops-team -s
...
+echo "7-Provision SQL & Mobile App  (bash ./provision_sql_mobileapp.sh -s ./test_fetch_build -g $resourceGroupTeam -l $resourceGroupLocation -q $sqlServerName -k $keyVaultName -u $sqlServerUsername -p $sqlServerPassword -d $sqlDBName)"
```
Comment (dtzar) [inline provision-team/setup.sh]:
```suggestion
echo "7-Provision SQL (bash ./provision_sql_mobileapp.sh -s ./test_fetch_build -g $resourceGroupTeam -l $resourceGroupLocation -q $sqlServerName -k $keyVaultName -u $sqlServerUsername -p $sqlServerPassword -d $sqlDBName)"
```

File: provision-team/setup.sh [hunk 3]
```diff
@@ -249,8 +245,10 @@ bash ./git_fetch.sh -u https://github.com/Azure-Samples/openhack-devops-team -s
...
+bash ./provision_sql_mobileapp.sh -g $resourceGroupTeam -l $resourceGroupLocation -q $sqlServerName -k $keyVaultName -u $sqlServerUsername -p $sqlServerPassword -d $sqlDBName
```
Comment (dtzar) [inline provision-team/setup.sh]: would be good to rename the file since it is no longer provisioning the mobileapp.

Comment (dcaro) [reply provision-team/setup.sh]: done

File: provision-team/setup.sh [hunk 4]
```diff
@@ -249,8 +245,8 @@ bash ./git_fetch.sh -u https://github.com/Azure-Samples/openhack-devops-team -s
 echo "6-Deploy ingress  (bash ./deploy_ingress_dns.sh -s ./test_fetch_build -l $resourceGroupLocation -n ${teamName}${teamNumber})"
 bash ./deploy_ingress_dns.sh -s ./test_fetch_build -l $resourceGroupLocation -n ${teamName}${teamNumber}
 
-echo "7-Provision SQL & Mobile App  (bash ./provision_sql_mobileapp.sh -s ./test_fetch_build -g $resourceGroupTeam -l $resourceGroupLocation -q $sqlServerName -m $mobileAppName -h $hostingPlanName -k $keyVaultName -u $sqlServerUsername -p $sqlServerPassword -d $sqlDBName)"
-bash ./provision_sql_mobileapp.sh -g $resourceGroupTeam -l $resourceGroupLocation -q $sqlServerName -m $mobileAppName -h $hostingPlanName -k $keyVaultName -u $sqlServerUsername -p $sqlServerPassword -d $sqlDBName
+echo "7-Provision SQL & Mobile App  (bash ./provision_sql.sh -s ./test_fetch_build -g $resourceGroupTeam -l $resourceGroupLocation -q $sqlServerName -k $keyVaultName -u $sqlServerUsername -p $sqlServerPassword -d $sqlDBName)"
```
Comment (dtzar) [inline provision-team/setup.sh]:
```suggestion
echo "7-Provision SQL (bash ./provision_sql.sh -s ./test_fetch_build -g $resourceGroupTeam -l $resourceGroupLocation -q $sqlServerName -k $keyVaultName -u $sqlServerUsername -p $sqlServerPassword -d $sqlDBName)"
```

**Pull request merged**